server/src/index.js

```diff
@@ -36,7 +36,10 @@ require('./services/watcher')
 
 Event.clients = Clients
 
-if (!config.has('devOptions.skipUpdateCheck')) {
+const skipUpdateCheck = config.has('devOptions.skipUpdateCheck')
+  ? config.getSafe('devOptions.skipUpdateCheck')
+  : false
+if (!skipUpdateCheck) {
```

Comment:
```suggestion
if (!config.getSafe('devOptions.skipUpdateCheck')) {
```
The only reason the `has` is being called was because this value often didn't exist in the config. Now that it's in default.json, you don't need it and you can just check the value right in the if statement.

Reply:
Updated, rebased, and retargeted to `develop` 👍